src/Idris/TypeSearch.hs

```diff
@@ -55,9 +55,13 @@ searchByType pkgs pterm = do
          displayScore theScore <> char ' ' <> prettyDocumentedIst i docInfo
                 | (n, theScore) <- names']
   case idris_outputmode i of
-    RawOutput _  -> do mapM_ iRenderOutput docs
+    RawOutput _  -> do if (not (null docs))
```

Comment:
I think the repeated conditional here is a bit inelegant, because the False case is duplicated. Why not do:
```
if (not (null docs))
  then case idris_outputmode i of ...
  else iRenderError $ text "No results found"
```